recipes/aster/build.sh

```diff
@@ -5,8 +5,9 @@ do
     mkdir -p "${PREFIX}/etc/conda/${CHANGE}.d"
     cp "${RECIPE_DIR}/${CHANGE}.sh" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
 done
-
-sed -i.bak1 's/-march=native/-march=x86-64 -mtune=generic/g' makefile
+if [ `uname -m` != "aarch64" ]; then
```

Comment:
```suggestion
if [ `uname -m` == "x86_64" ]; then
```
IMO this would be better. This way it could work on ppc64le later if more architectures are added.